collections/ansible_collections/massopencloud/esi/roles/network/meta/argument_specs.yaml

```
@@ -0,0 +1,15 @@
+argument_specs:
+  main:
+    options:
+      network_state:
+        type: "str"
+        required: true
+        choices:
+        - present
+        - absent
+      network_name_or_id:
```

```suggestion
      network_name_or_id:
      description: >
         Name of the network in case of network when state is `present`.
         Name or ID of the network when state is `absent`, the deletion will fail if the network name is not unique.
```

Add some description in case we go with `network_name_or_id` and remove the use of `network_name`

---

Done!